File: 프로그래머스/1/176963. 추억 점수/추억 점수.py
def solution(name, yearning, photo):
    answer = [0] * len(photo)
    for r in range(len(photo)):
        for c in range(len(photo[r])):
            for i in range(len(name)):
                if(photo[r][c] == name[i]):
                    answer[r] += yearning[i]
    return answer

File: 프로그래머스/1/176963. 추억 점수/추억 점수.java
class Solution {
    public int[] solution(String[] name, int[] yearning, String[][] photo) {
        int[] answer = {};
        answer = new int[photo.length];
        for (int r=0; r<photo.length; r++){
            for (int c=0; c<photo[r].length; c++){
                for (int i=0; i<name.length; i++){
                    if(photo[r][c].equals(name[i])){
                        answer[r] += yearning[i];
                    }
                }
            }
        }
        return answer;
    }
}